config/nuttx/stm32f4dis/app/setjmp.S

```diff
@@ -0,0 +1,65 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
Does this file come from JerryScript ? I wonder whether we should change the license clauses or not.